src/20231116_AaveV3Ethereum_CRVUSDOnboardingOnAaveV3Ethereum/AaveV3Ethereum_CRVUSDOnboardingOnAaveV3Ethereum_20231116.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumEModes} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum} from 'aave-helpers/v3-config-engine/AaveV3PayloadEthereum.sol';
+import {EngineFlags} from 'aave-helpers/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-helpers/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IV3RateStrategyFactory} from 'aave-helpers/v3-config-engine/IV3RateStrategyFactory.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+
+/**
+ * @title CRVUSD onboarding on Aave V3 Ethereum
+ * @author @Marczeller - Aave-chan Initiative
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xbc10b43fccd3954f02c9df774ba6f8335268727b999660738ae37a1b9d5b969e
+ * - Discussion: https://governance.aave.com/t/arfc-crvusd-onboarding-on-aave-v3-ethereum-pool/15161
+ */
+contract AaveV3Ethereum_CRVUSDOnboardingOnAaveV3Ethereum_20231116 is AaveV3PayloadEthereum {
+  using SafeERC20 for IERC20;
+
+  address public constant crvUSD = 0xf939E0A03FB07F59A73314E73794Be0E57ac1b4E;
+  uint256 internal constant crvUSD_SEED_AMOUNT = 11 ** 18;
```

Comment:
I think here you meant the amount to be 11 with 18 decimals. if so, you need to do it like this 11e18 or 11 ether
11 ** 18 is 5559917313492231481

And better if you make it public so it can be checked externally

Reply:
Solved